ocs_ci/ocs/ui/mcg_ui.py

```diff
@@ -345,6 +402,9 @@ def create_obc_ui(self, obc_name, storageclass, bucketclass=None):
         logger.info("Select Storage Class")
         self.do_click(self.obc_loc["storageclass_dropdown"])
         self.do_send_keys(self.obc_loc["storageclass_text_field"], storageclass)
+
+        from ocs_ci.ocs.ui.helpers_ui import format_locator
+
```

Comment:
Can we move the import to the top?

Reply:
Importing it from global creates an import loop, since helpers_ui imports BaseUi